python/paddle/v2/framework/tests/test_sum_op.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+import numpy as np
+from op_test import OpTest
+import paddle.v2.framework.core as core
+
+
+class TestSumOp(OpTest):
+    def setUp(self):
+        self.op_type = "sum"
+        x0 = np.random.random((3, 4)).astype('float32')
+        x1 = np.random.random((3, 4)).astype('float32')
+        x2 = np.random.random((3, 4)).astype('float32')
+        self.inputs = {"X": {"x0": x0, "x1": x1, "x2": x2}}
```

Comment:
I think maybe the user does not want to specify names about `x0`, `x1`, `x2`.

Just,

```python
self.inputs = {
  "X": [x0, x1, x2]
}
```

is OK.

Reply:
> Just,

> self.inputs = {
>   "X": [x0, x1, x2]
> }
> is OK.

If so, how to generate the multi-inputs name for "x0", "x1" and  "x2"?  Generate these names automatically according to 'X'?

Reply:
@reyoung @qingqing01 In grad check process, we can chose a specific input. Such as 
```
self.check_grad(["x0"], "Out")
```

So, explicitly set a name is more flexible.